test/geochem/geochem_alteration.py

```diff
@@ -87,5 +87,39 @@ def test_WIP(self):
         df.loc[:, "WIP"] = WIP(df)
 
 
+if __name__ == "__main__":
```

Comment:
We only need the `unittest.main()` call once per file, and there should already be one at the end, so feel free to remove the extras here.